api/explore/explore.js

```diff
@@ -39,24 +52,26 @@ async function GetBoxes(size, dateString, request, amount) {
   const nextHoliday = holiday(dateString);
   const boxes = [];
   retval.explore = [];
+  const min = 10;
```

Comment:
what are you trying to accomplish?

Reply:
Not returning a box with too little recipes in it.